Gemfile

```diff
@@ -5,6 +5,7 @@ gem "bcrypt", "3.1.11"
 gem "coffee-rails", "4.2.2"
 gem "config"
 gem "faker", "1.7.3"
+gem "figaro"
```

Comment:
em thấy 2 thằng này nó kiểu same same nhau, chắc là tùy nhu cầu công việc và SỞ THÍCH CÁ NHÂN :D vì theo em tìm hiểu thì đây là cách người viết ra figaro nói vậy :D 
https://github.com/laserlemon/figaro#is-figaro-like-dotenv